serving/docker/lmi.Dockerfile

```diff
@@ -81,6 +81,7 @@ RUN apt-get update && apt-get install -yq libaio-dev libopenmpi-dev g++ unzip cu
 COPY requirements-lmi.txt ./requirements.txt
 RUN pip3 install torch==2.5.1 torchvision==0.20.1 --index-url https://download.pytorch.org/whl/cu124 && pip3 cache purge
 RUN pip3 install -r requirements.txt \
+    && pip3 install onnxruntime-gpu==$djl_onnx_version \
```

Comment:
Can we add this to the requirements-lmi.txt file?

Reply:
To make onnxruntime-gpu to work, it needs to be installed after onnxruntime is installed. I found that if I put it in requirements-lmi.txt, the order is not guaranteed.

Reply:
Do we need to install both onnxruntime and onnxruntime-gpu? Could we replace onnxruntime in the requirements file with onnxruntime-gpu==1.20.0?

Reply:
We need both onnxruntime and onnxruntime-gpu

Reply:
1. djl_converter will install onnxruntime as a dependency. I have moved djl_converter out of requirements-lmi.txt with --no-deps.
2. Moved onnxruntime-gpu into requirements-lmi.txt